### ISSUE TITLE
Build fails with clang-4.0

### ISSUE THREAD

**Issue by yurivict** [abau/dilay#37]
Error:
```
src/tool/trim-mesh/action.cpp:348:27: error: constexpr variable 'bestAngle' must be initialized by a constant expression
          constexpr float bestAngle = glm::cos (glm::radians (60.0f));
                          ^           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

There are also warnings:
```
src/camera.cpp:16:1: warning: 'Impl' defined as a struct here but previously declared as a class [-Wmismatched-tags]
struct Camera::Impl
^
src/camera.hpp:47:3: note: did you mean struct here?
  IMPLEMENTATION
  ^
src/macro.hpp:776:3: note: expanded from macro 'IMPLEMENTATION'
  class Impl;          \
  ^
src/dynamic/octree.cpp:43:3: warning: 'IndexOctreeNode' defined as a struct here but previously declared as a class [-Wmismatched-tags]
  struct IndexOctreeNode
  ^
src/dynamic/octree.cpp:27:3: note: did you mean struct here?
  class IndexOctreeNode;
  ^~~~~
  struct

src/history.cpp:120:28: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
    this->past.push_front (std::move (sceneSnapshot (scene, config)));
                           ^
src/history.cpp:120:28: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
    this->past.push_front (std::move (sceneSnapshot (scene, config)));
                           ^
src/history.cpp:120:28: note: remove std::move call here
    this->past.push_front (std::move (sceneSnapshot (scene, config)));
                           ^~~~~~~~~~~                             ~
src/history.cpp:145:32: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
      this->future.push_front (std::move (sceneSnapshot (state.scene (), config)));
                               ^
src/history.cpp:145:32: note: remove std::move call here
      this->future.push_front (std::move (sceneSnapshot (state.scene (), config)));
                               ^~~~~~~~~~~                                      ~
src/history.cpp:157:30: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
      this->past.push_front (std::move (sceneSnapshot (state.scene (), config)));
                             ^
src/history.cpp:157:30: note: remove std::move call here
      this->past.push_front (std::move (sceneSnapshot (state.scene (), config)));
                             ^~~~~~~~~~~                                      ~
src/sketch/conversion.cpp:914:18: warning: variable 'iu' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
        else if (edge == 2)
                 ^~~~~~~~~
src/sketch/conversion.cpp:920:40: note: uninitialized use occurs here
        makeQuad (edge, s1 >= 0.0f, i, iu, iv, iuv);
                                       ^~
src/sketch/conversion.cpp:914:14: note: remove the 'if' if its condition is always true
        else if (edge == 2)
             ^~~~~~~~~~~~~~
src/sketch/conversion.cpp:901:24: note: initialize the variable 'iu' to silence this warning
        unsigned int iu, iv, iuv;
                       ^
                        = 0
src/sketch/conversion.cpp:914:18: warning: variable 'iv' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
        else if (edge == 2)
                 ^~~~~~~~~
src/sketch/conversion.cpp:920:44: note: uninitialized use occurs here
        makeQuad (edge, s1 >= 0.0f, i, iu, iv, iuv);
                                           ^~
src/sketch/conversion.cpp:914:14: note: remove the 'if' if its condition is always true
        else if (edge == 2)
             ^~~~~~~~~~~~~~
src/sketch/conversion.cpp:901:28: note: initialize the variable 'iv' to silence this warning
        unsigned int iu, iv, iuv;
                           ^
                            = 0
src/sketch/conversion.cpp:914:18: warning: variable 'iuv' is used uninitialized whenever 'if' condition is false [-Wsometimes-uninitialized]
        else if (edge == 2)
                 ^~~~~~~~~
src/sketch/conversion.cpp:920:48: note: uninitialized use occurs here
        makeQuad (edge, s1 >= 0.0f, i, iu, iv, iuv);
                                               ^~~
src/sketch/conversion.cpp:914:14: note: remove the 'if' if its condition is always true
        else if (edge == 2)
             ^~~~~~~~~~~~~~
src/sketch/conversion.cpp:901:33: note: initialize the variable 'iuv' to silence this warning
        unsigned int iu, iv, iuv;
                                ^
                                 = 0

src/tool/sculpt/util/brush.hpp:86:8: warning: 'sculpt' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  void sculpt (const SculptBrush&, const DynamicFaces&) const;
       ^
src/tool/sculpt/util/brush.hpp:35:16: note: overridden virtual function is here
  virtual void sculpt (const SculptBrush&, const DynamicFaces&) const = 0;
               ^
src/tool/sculpt/util/brush.hpp:90:31: warning: 'discardBack' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  MEMBER_GETTER_SETTER (bool, discardBack);
                              ^
src/tool/sculpt/util/brush.hpp:29:16: note: overridden virtual function is here
  virtual bool discardBack () const { return true; }
               ^
src/tool/sculpt/util/brush.hpp:111:8: warning: 'sculpt' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  void sculpt (const SculptBrush&, const DynamicFaces&) const;
       ^
src/tool/sculpt/util/brush.hpp:35:16: note: overridden virtual function is here
  virtual void sculpt (const SculptBrush&, const DynamicFaces&) const = 0;
               ^
src/tool/sculpt/util/brush.hpp:119:8: warning: 'sculpt' overrides a member function but is not marked 'override' [-Winconsistent-missing-override]
  void sculpt (const SculptBrush&, const DynamicFaces&) const;
       ^
src/tool/sculpt/util/brush.hpp:35:16: note: overridden virtual function is here
  virtual void sculpt (const SculptBrush&, const DynamicFaces&) const = 0;
               ^

```

**Comment by abau** [comment]
These are already fixed in the latest master.

**Comment by yurivict** [comment]
Could you make a release?

**Comment by abau** [comment]
Yes, I will in a day or two when I fixed #25.

**Comment by abau** [comment]
> Could you make a release?

https://github.com/abau/dilay/releases/tag/1.6.1
